targets/TARGET_STM/TARGET_STM32F4/TARGET_UBLOX_EVK_ODIN_W2/sdk/ublox-odin-w2-drivers/bt_types.h

```diff
@@ -0,0 +1,193 @@
+/*---------------------------------------------------------------------------
```

Comment:
I don't think we can get that 'license' in. You should license your headers the same way you license your binary that is PBL.